### PR TITLE
JavaScript Group Capstone

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, we did the following:
- Set up API
- Setup Webpack and Jest
- Install linter files
- Used Gitflow

### CODE REVIEW

**Review by o-rashad — CHANGES_REQUESTED** [review]
Asslam Alikom Team 

Hi 👋🏽, 
While you made a great effort in this project, unfortunately, 
I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a capstone project, 
1. `That doesn't include your video presentation, please provide a video recording according to the guidelines and put it in the PR description or README file`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const row = document.querySelector('.row');
+const urlItems = 'https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood';
+
+/* Generate urls of the involvement API */
+const urlStart = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const apiId = 't9up9m5T6uECqwieuC71';
+const urlLike = `${urlStart + apiId}/likes`;
+// const urlComment = `${urlStart + apiId}/comments`;
+// const urlReservation = `${urlStart + apiId}/reservations`;
```
Comment (CollinsTatang) [inline src/index.js:17]: - [ ] Please remove commented code out of your codebase.

File: src/modules/displayMain.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import getData from './getData.js';
+import { getLikes } from './likes.js';
+// import itemCount from './itemCounter.js'; //Try to fix this implementation please guys
+
+export default async function displayItems(mealContainer, url, urlLike) {
+  let divMeal = '';
+  const likeList = await getLikes(urlLike);
+  const data = await getData(url); // Copy this line to get your data
+  // const count = itemCount(data);
+  // document.querySelector('.nav-item').children[0].innerHTML += `(${count})`;
```
Comment (CollinsTatang) [inline src/modules/displayMain.js:11]: - [ ] Please just do as mention in the above comment

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @moise-mulungu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Please add the `home page footer` similar to the [mockup](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**